.github/workflows/argilla-frontend.deploy-environment.yml

```
@@ -63,7 +63,7 @@ jobs:
             ADMIN_API_KEY=${{ steps.credentials.outputs.admin }}
             ANNOTATOR_PASSWORD=${{ steps.credentials.outputs.annotator }}
             ANNOTATOR_API_KEY=${{ steps.credentials.outputs.annotator }}
-            ARGILLA_ENABLE_TELEMETRY=0
+            HF_HUB_DISABLE_TELEMETRY=1
```

It's easier to use affirmative settings:

```suggestion
            HF_HUB_ENABLE_TELEMETRY=0
```

In fact we already have an `ARGILLA_ENABLE_TELEMETRY` environment variable.

---

@jfcalvo, I know but I think it is cleaner to default to the Hugging Face one because everything is stored there and to default to other HF-related configs people might have w.r.t. telemetry.